nRF52832 hello-world.flash unknown rule - How to flash it?

Hello,

i like to flash the hello-world example on my nRF52823 over USB.
Is there an other way to flash it with Contiki-**NG**?
I follow the tutorial steps from Contiki-**OS**.:

- https://github.com/contiki-os/contiki/tree/master/platform/nrf52dk

But the command "_make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 hello-world.flash_"
produce this error in Contiki-NG:

```
SDK: /home/pi/nrf5x-sdk2
SoftDevice: /home/pi/nrf5x-sdk2/components/softdevice/s1xx_iot/s1xx-iot-prototype3_nrf52_softdevice.hex
make: *** Keine Regel, um „hello-world.flash“ zu erstellen.  Schluss.
```

**My commands**
1. sudo su 
2. export NRF52_SDK_ROOT=/home/pi/nrf5x-sdk2
3. export NRF52_JLINK_PATH=/home/pi/JLink_Linux_V640_arm/
4. make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 clean
5. make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 erase
6. make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 softdevice.flash
7. make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 hello-world.flash --> ERROR
(8. make TARGET=nrf52dk BOARD=nrf52832 PORT=/dev/ttyACM0 hello-world)

**With Contiki-OS it works fine. How can i flash it with Contiki-NG? Thanks!**

--- a/arch/cpu/nrf52832/Makefile.nrf52832
+++ b/arch/cpu/nrf52832/Makefile.nrf52832
@@ -185,10 +185,12 @@ $(OBJECT_DIRECTORY)/%.o: %.s
 	$(TRACE_CC)
 	$(Q)$(CC) $(ASMFLAGS) $(addprefix -I$(NRF52_SDK_ROOT)/, $(INC_PATHS)) -c -o $@ $<
 
-%.jlink:
-	sed -e 's/#OUTPUT_FILENAME#/$*.hex/' $(CONTIKI_CPU)/flash.jlink > $@
+include $(ARCH_PATH)/cpu/arm/cortex-m/cm4/Makefile.cm4
 
-%.flash: %.hex %.jlink
+%.jlink: $(OUT_HEX)
+	sed -e 's,#OUTPUT_FILENAME#,$<,' $(CONTIKI_CPU)/flash.jlink > $@
+
+%.flash: %.jlink
 	@echo Flashing: $^
 	$(JLINK) $(JLINK_OPTS) -CommanderScript $*.jlink
 
@@ -203,5 +205,3 @@ erase:
 	$(JLINK) $(JLINK_OPTS) -CommanderScript $(CONTIKI_CPU)/erase.jlink
 
 .PHONY: softdevice.jlink
-
-include $(ARCH_PATH)/cpu/arm/cortex-m/cm4/Makefile.cm4

